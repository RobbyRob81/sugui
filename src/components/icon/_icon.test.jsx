import * as React from 'react';
import { customRender } from '../../../test/test-utils';
import Icon from '.';

it('should render without throwing an error', () => {
  const { container } = customRender(<Icon />);
  expect(container.firstChild).toMatchSnapshot();
});
