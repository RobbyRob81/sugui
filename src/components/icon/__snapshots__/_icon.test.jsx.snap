// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render without throwing an error 1`] = `
<svg
  class="sc-bdVaJa fhdTuk"
  viewBox="0 0 20 20"
>
  <use
    xlink:href="#"
  />
</svg>
`;
